cluster.tf

```diff
@@ -1,104 +1,78 @@
-##############################################################################
-# Find valid IKS/ROKS Cluster versions for region
-##############################################################################
-
-data "ibm_container_cluster_versions" "cluster_versions" {}
-
-##############################################################################
-
-
 ##############################################################################
```

Comment:
The logic related to creating iks cluster should remain here. The module should be used for ocp clusters only